modules/nixos/services/bobaboard/default.nix

```diff
@@ -12,7 +13,9 @@ in
   };
   config = mkIf cfg.enable {
     networking.firewall.allowedTCPPorts = [
-      4200
+      80
+      443
+      6900
```

Comment:
These are fine for now, but wanted to call out that the ports are typically user-defined per-system and the backend address should be configurable.

These are things we can address in the future 👍